### DISCUSSION TITLE
I found that the Merge shapes/tracks function is not available, can someone fix this function? Thank you

### DISCUSSION THREAD

**Discussion by yuchiahui0530** [cvat-ai/cvat#3592] [Q&A] [answered]
I found that the Merge shapes/tracks function is not available, can someone fix this function? Thank you

**Comment by bsekachev** [comment, marked as answer]
Hi @yuchiahui0530 

Why do you think it is unavailable?

**Reply by yuchiahui0530** [reply]
Hello, I tried to use two boxes to merge, but it couldn't be used. I also read the function instructions to operate, but it still can’t be used. I asked my colleague to test and I can’t use the two boxes to merge. Thank you

**Reply by bsekachev** [reply]
Are you trying to merge boxes from different frames and with the same label?

**Reply by yuchiahui0530** [reply]
Thank you for your reply, I just tried the method you said, but still can’t merge. Thank you

**Reply by bsekachev** [reply]
Could you please attach a gif? I've just tried merge on cvat.org and it works as expected.

**Reply by yuchiahui0530** [reply]
Sir. I just sent the operation video to your mail, boris.sekachev@intel.com, thank you

**Reply by bsekachev** [reply]
Okay, got it. When you use merge, you can select for merge only one object on one frame.

So, you can select the first object on the frame 0, than go to frame 1 and select the second object (agree, not very obvious when merging tracks which intersect each other between frames). 
Ideally objects to merge should not intersect on frames. When the object ends, you should mark it as "outside" (it disappear on current and next frames), then merge.

**Reply by yuchiahui0530** [reply]
Sir  Sorry, I can’t understand what you mean, I just did it once and I couldn’t merge

**Reply by bsekachev** [reply]
You should not merge tracks ``A`` which is on frames [0:end] and ``B``  which is on frames [1:end], because they have intersected frames [1:end]. If you need to merge them, first mark track ``A`` as "outside" ([refer to documentation](https://openvinotoolkit.github.io/cvat/docs/manual/basics/track-mode-basics/)) on frame 1, and then merge these tracks.

**Reply by bsekachev** [reply]
As it was mentioned in my first reply, it is important that objects for merge should be located on ``different`` frames. On the video you are trying to merge two objects on the same frame.

**Reply by yuchiahui0530** [reply]
I already know how to merge, thank you very much for your help
